scripts/flu_hosp_prod.R

```diff
@@ -99,24 +100,14 @@ rlang::list2(
         select(-version) %>%
         data_substitutions(disease = "flu")
     },
-    cue = tar_cue(mode = "always")
+    cue = targets::tar_cue(mode = "thorough")
```

Comment:
"thorough" is default, yea? https://docs.ropensci.org/targets/reference/tar_cue.html?q=thorough#ref-usage

Reply:
yeah. Things are always getting redone b/c the `forecasters` target is always getting redone for some reason.